### PR TITLE
feature: Enhanced error message for unknown errors (not in prod mode)

### PR DESCRIPTION
**Pull request by tbarlow12**

### CODE REVIEW

File: src/registerToolbar.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export default function registerToolbar() {
         icon: "fa-trash-alt",
         group: ToolbarItemGroup.Regions,
         type: ToolbarItemType.Action,
+        accelerators: ["Ctrl+Delete"],
```
Comment (mydiemho) [inline src/registerToolbar.ts]: should put this in a different PR, I prefer not mixing code

File: src/react/components/common/errorHandler/errorHandler.tsx [hunk 2]
```diff
@@ -79,7 +79,6 @@ export class ErrorHandler extends React.Component<IErrorHandlerProps> {
     private handleError(error: string | Error | AppError) {
         console.log(error);
```
Comment (mydiemho) [inline src/react/components/common/errorHandler/errorHandler.tsx]: we should also remove this left over code

File: src/react/components/common/errorHandler/errorHandler.tsx [hunk 3]
```diff
@@ -103,29 +102,40 @@ export class ErrorHandler extends React.Component<IErrorHandlerProps> {
                 message: reason.message,
                 title: reason.name,
             };
+        } else {
+            appError = {
```
Comment (mydiemho) [inline src/react/components/common/errorHandler/errorHandler.tsx]: I think we can combine this with the previous case.  sometimes it happens that type Error has no message or name

File: src/registerToolbar.ts (hunk 1 above)
Comment (tbarlow12) [reply src/registerToolbar.ts]: Cool. Will do

File: src/react/components/common/errorHandler/errorHandler.tsx [hunk 4]
```diff
@@ -103,29 +102,40 @@ export class ErrorHandler extends React.Component<IErrorHandlerProps> {
...
+                message: this.getUnknownErrorMessage(error),
+            };
         }
+        this.props.onError(appError);
+    }
 
-        if (!appError) {
-            appError = new AppError(ErrorCode.Unknown, "Unknown Error occurred");
+    private getUnknownErrorMessage(e) {
+        if (Env.get() !== "production") {
```
Comment (wbreza) [inline src/react/components/common/errorHandler/errorHandler.tsx]: I'm assuming when we do a prod build `create-react-app` sets node_env to "production" ?

Comment (tbarlow12) [reply src/react/components/common/errorHandler/errorHandler.tsx]: That's what I understood. I could be wrong, but I'll look into this further

Comment (tbarlow12) [reply src/react/components/common/errorHandler/errorHandler.tsx]: Verified this with a release version.

**Comment by tbarlow12** [comment]
Closing in favor of [PR against dev branch](https://github.com/Microsoft/VoTT/pull/639)
